File: Assignment_3/Assignment3_1.py
"""-------------------------------------------------------------------------------------------
                          Assignment3_1
                    (Student name - Vaishali Jorwekar)
                     Python by Marvellous Infosystems
---------------------------------------------------------------------------------------------
Problem statement : This program accepts N elements from the user and 
                    store it into a list and returns the addition of 
                    elements of the list
------------------------------------------------------------------------------------------"""
from MarvellousFun import acceptElements
#-------------------------------------------------------------------------------------
# This Function adds all the elements from input list
#-------------------------------------------------------------------------------------
def additionOfAllElements(inputNumberList):
    sum=0
    for num in inputNumberList:
         sum= sum + num
    print("-------------------------------------------------------------------")
    print(f"Addition of all list elements {inputNumberList} is : {sum}")     
    print("-------------------------------------------------------------------")

#------------------------------------------------------------------------------------------
# This function accepts a number of elements number from user and calls function 
# acceptElements(number) to accept elements in the list 
#------------------------------------------------------------------------------------------

def main():
    noOfElements=int(input("Enter number of elements in the list:"))
    inputNumberList = acceptElements(noOfElements)
    additionOfAllElements(inputNumberList)
   
#-----------------------------------------------------------------------------------
# This is main entry point of the program
#-----------------------------------------------------------------------------------
if __name__=="__main__":
    main()
#-----------------------------------------------------------------------------------

File: JavaPractice/Assignment_3/Assignment3_1.java
/*-------------------------------------------------------------------------------------------
                          Assignment3_1
                    (Student name - Vaishali Jorwekar)
                     Python by Marvellous Infosystems
---------------------------------------------------------------------------------------------
Problem statement : This program accepts N elements from the user and 
                    store it into a list and returns the addition of 
                    elements of the list
-----------------------------------------------------------------*/

import java.util.ArrayList;

public class Assignment3_1 {
	
	public static void main(String args[]) {
		
		
		ArrayList<Integer> numberList=Module.acceptNumbers();
		System.out.println("List Elements are:"+numberList);
		
		
		sumListElements(numberList);
		
		
	}

	private static void sumListElements(ArrayList<Integer> numberList) {
		int sumElements=0;
		for(int cnt=0;cnt<numberList.size();cnt++)
			sumElements=sumElements+numberList.get(cnt);
		
		System.out.println("Sum Of Elements is :"+sumElements);
	}

}
